README.md

```diff
@@ -0,0 +1,209 @@
+
+After the proposal is accepted or declined (whether after step 2 or step 4),
+implementation work proceeds in the same way as any other contribution.
+
+## How to write a proposal
+
+The issue titles must follow this schema:
+
+```    
+    proposal: <domain>: <title>
+```
+
+The `domain` refers to the project related to the proposal\
+The `title` should be short and focused on the main proposal topic.
+
+Issue descriptions must be clear and relevant to the core idea expressed in the title.
+
+Proposal issues must be accompanied by the '*proposal*' tag, making them easier to track on issue pages
+
+## Design Documents
+
+As noted above, when needed proposals can be deeper elaborated in a design document.
+
+- The design doc should be added to the repository with a pull request to the `main` branch in [this current](https://github.com/web-of-things-open-source/proposal) repository
+- The pull request should come from a branch whose name is composed as:
+```
+<Issue number>-<domain of the issue>-<short proposal title>
+```
+- The doc should be inserted into the `/design` folder, following the same schema above:
+```
+<Issue number>-<domain of the issue>-<short proposal title>.md
+```
+- if the document has media files, code examples or other files that are part of the documentation, they must be uploaded in a folder inside the `/design` path. The folder has the same name as the documentation file 
+
+- The documentation must be written following [this template](./TEMPLATE.md).
+
+- [Each sentence should start on a new line](http://rhodesmill.org/brandon/2012/one-sentence-per-line/)
+so that comments can be made accurately and the diff kept shorter.
+
+#### Example of a documentation upload
+
+The user Mario made a proposal and now is asked to upload documentation related to it to deepen the analysis on the topic.
+
+- *issue* -> "proposal: foo: Change bar api path" 
+- *issue number* -> #42069
+- *Documentation structure*:
+```bash
+/
+└── design/
+    ├── 42069-foo-change-bar-path.md
+    └── 42069-foo-change-bar-path/
+        ├── image-1.png
+        ├── image-2.png
+        └── code example.go
+```
+
+## Proposals Review
+
+A group of the wot team holds “proposals review meetings”
+periodically. 
+The group will review the proposals and discuss the changes 
+that can be advanced or declined.
+The review is soley based on the engagement and general consensus under each issue discussion.
+
+From each review meeting the team will change the status of the observed proposals to reflect the community decisions regarding them. This change will be noted on the corresponding issue pages by the team itself. 
+
+Proposal issues are tracked in the
+[Proposals project](https://github.com/orgs/web-of-things-open-source/projects/1/views/1) page where they are sorted by status.
+
+All the review meetings will be documented in form of *minutes* collected inside the [minutes repository](https://github.com/web-of-things-open-source/minutes)
+
```

Comment:
I know we talked about this, but I looked at the golang repo again and the actual comments are always created under the related issue. What was the reason for a separate repo again? The golang meeting minutes merely list the discussed issues and the proposal status. Do you expect more to be put into the meeting minutes?

Reply:
I would use a repo instead of issues to organize better the minutes. The single issue is really big, and a little bit tedious to look at if you search for past minutes.